pilot/pkg/networking/core/v1alpha3/tracing.go

```diff
@@ -315,8 +317,37 @@ func convert(ctxs []meshconfig.MeshConfig_ExtensionProvider_OpenCensusAgentTraci
 	return converted
 }
 
+func dryRunPolicyTraceTag(name, key string) *tracing.CustomTag {
+	return &tracing.CustomTag{
```

Comment:
please document that the tag will not be populated when not used as there is no default value set for the tag (and maybe linking to the envoy docs). if the underlying envoy behavior changes, this could impact users.

Reply:
Added comments, thanks.